scripts/lib/dbsync_healthcheck.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 export PGPASSWORD=${POSTGRES_PASSWORD}
-[[ $(( $(date +%s) - $(date --date="$(psql -U ${POSTGRES_USER} -d ${POSTGRES_DB} -qt -c 'select time from block order by id desc limit 1;')" +%s) )) -lt 3600 ]] || exit 1
+[[ $(( $(date +%s) - $(date --date="$(psql -U ${POSTGRES_USER} -d ${POSTGRES_DB} -h ${POSTGRES_HOST} -qt -c 'select time from block order by id desc limit 1;')" +%s) )) -lt 3600 ]] || exit 1
```

Comment:
This shouldnt be required (same as Port) since the env variables via compose already specify PGHOST and PGPORT